pkg/reconciler/instances/istio/get/get.go

```diff
@@ -0,0 +1,88 @@
+package get
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/kyma-incubator/reconciler/pkg/reconciler/chart"
+	"github.com/kyma-incubator/reconciler/pkg/reconciler/kubernetes"
+	"github.com/kyma-incubator/reconciler/pkg/reconciler/service"
+	"github.com/pkg/errors"
+	yaml3 "gopkg.in/yaml.v3"
+	"io"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"net/http"
+)
+
+// The kind is used for easy access to the configuration in Kyma chart
+const configurationKind = "IstioOperatorConfiguration"
+
+func IstioTagFromContext(context *service.ActionContext) (string, error) {
```

Comment:
we do not have unit test for this? there is some logic we should cover with a test.